--- scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
+++ scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
@@ -2644,7 +2644,7 @@
     },
     {
         name        => "ZM_CHECK_FOR_UPDATES",
-        default     => "yes",
+        default     => "no",
         description => "Check with zoneminder.com for updated versions",
         help        => qqq("
             From ZoneMinder version 1.17.0 onwards new versions are
