--- scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in	2015-06-21 19:38:54.802933728 +0100
+++ scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm_new.in	2015-06-21 20:02:16.174947458 +0100
@@ -2644,7 +2644,7 @@
     },
     {
         name        => "ZM_CHECK_FOR_UPDATES",
-        default     => "yes",
+        default     => "no",
         description => "Check with zoneminder.com for updated versions",
         help        => qqq("
             From ZoneMinder version 1.17.0 onwards new versions are
