--- scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
+++ scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
@@ -915,9 +915,9 @@
     },
     {
         name        => "ZM_PATH_FFMPEG",
-        default     => "@PATH_FFMPEG@",
+        default     => "/usr/bin/avconv",
         description => "Path to (optional) ffmpeg mpeg encoder",
-        help        => "This path should point to where ffmpeg has been installed.",
+        help        => "This path should point to where ffmpeg has been installed.  Note that this installation expects avconv to satisfy ffmpeg functionality",
         requires    => [ { name=>"ZM_OPT_FFMPEG", value=>"yes" } ],
         type        => $types{abs_path},
         category    => "images",
