Last-Update: 2015-03-30
Forwarded: https://github.com/ZoneMinder/ZoneMinder/pull/759
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: POD added missing NAME sections

--- a/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8608W_Y2k.pm
+++ b/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8608W_Y2k.pm
@@ -657,11 +657,11 @@
 1;
 __END__
 # Below is stub documentation for your module. You'd better edit it!
 
-=head1 FI-8608W
+=head1 NAME
 
-ZoneMinder::Database - Perl extension for FOSCAM FI-8608W by Christophe_Y2k
+ZoneMinder::Control::FI-8608W - Perl extension for FOSCAM FI-8608W by Christophe_Y2k
 
 =head1 SYNOPSIS
 
   use ZoneMinder::Database;
--- a/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8620_Y2k.pm
+++ b/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8620_Y2k.pm
@@ -733,11 +733,11 @@
 1;
 __END__
 # Below is stub documentation for your module. You'd better edit it!
 
-=head1 FI8620
+=head1 NAME
 
-ZoneMinder::Database - Perl extension for FOSCAM FI8620
+ZoneMinder::Control::FI8620 - Perl extension for FOSCAM FI8620
 
 =head1 SYNOPSIS
 
   use ZoneMinder::Database;
--- a/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8908W.pm
+++ b/scripts/ZoneMinder/lib/ZoneMinder/Control/FI8908W.pm
@@ -219,8 +219,12 @@
 
 __END__
 =pod
 
+=head1 NAME
+
+ZoneMinder::Control::FI8908W - Foscam FI8908W camera control
+
 =head1 DESCRIPTION
 
 This module contains the implementation of the Foscam FI8908W / FI8918W IP camera control
 protocol.
--- a/scripts/ZoneMinder/lib/ZoneMinder/Control/FI9821W_Y2k.pm
+++ b/scripts/ZoneMinder/lib/ZoneMinder/Control/FI9821W_Y2k.pm
@@ -695,11 +695,11 @@
 1;
 __END__
 # Below is stub documentation for your module. You'd better edit it!
 
-=head1 FI9821W
+=head1 NAME
 
-ZoneMinder::Database - Perl extension for FOSCAM FI9821W
+ZoneMinder::Control::FI9821W - Perl extension for FOSCAM FI9821W
 
 =head1 SYNOPSIS
 
   use ZoneMinder::Database;
